.github/workflows/doc-index-updater-branch.yaml

```diff
@@ -79,11 +79,16 @@ jobs:
           cargo build --release
           cargo test --release
 
+      - name: Create Docker tag
+        working-directory: products/medicines/doc-index-updater
+        run: |
+          TAG="$(git rev-parse --short=7 ${{ github.sha }})"
+          echo "TAG=$TAG" >>$GITHUB_ENV
+
       - name: Build and push Docker image
         if: steps.filter.outputs.src == 'true'
         working-directory: products/medicines/doc-index-updater
         run: |
-          TAG="$(git rev-parse --short=7 ${{ github.sha }})"
           make ci-master tag=$TAG image=$IMAGE
           DIGEST="$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE}:${TAG})"
           echo "TAG=$TAG" >>$GITHUB_ENV
```

Comment:
Can remove this line in each of them as line 86 tackles this!

Reply:
Good catch, I missed